backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/statement/QuizAnswerItem.java

```diff
@@ -44,8 +49,12 @@ public QuizAnswerItem() {
     public QuizAnswerItem(StatementQuizDto statementQuizDto) {
         this.quizId = statementQuizDto.getId();
         this.quizAnswerId = statementQuizDto.getQuizAnswerId();
-        Gson gson = new Gson();
-        this.answers = gson.toJson(statementQuizDto.getAnswers());;
+        ObjectMapper obj = new ObjectMapper();
```

Comment:
Caution about this change when it goes to production because there may exist data in the database compressed using Gson